apps/homepage/src/app/[lang]/home.module.css

```diff
@@ -1,8 +1,30 @@
-.main {
-  min-height: 100vh;
+.hero {
+  min-height: calc(100vh - 70px);
+  text-align: center;
 }
 
-.hero {
+.wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 2rem;
   text-align: center;
-  padding-top: 200px;
-}
\ No newline at end of file
+  gap: 1rem;
+  font-weight: 900;
+
+  &.isMobile {
+    font-size: 1rem;
```

Comment:
You could use `@uireact/text` and use a `size` prop to set this automatically to your theme's property